src/test/resources/testng.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="MyTestSuite">
```

Comment:
Top-level parallel="tests": Ensures ChromeTests and FirefoxTests run in parallel.